### PR TITLE
fix gce cloud provider projects api endpoint

### PR DESCRIPTION
**Pull request by freehan**
```release-note
NONE
```

fix bug introduced in https://github.com/kubernetes/kubernetes/pull/48642

### CODE REVIEW

**Comment by nicksardo** [comment]
lgtm, but will let @bowei take a look

File: pkg/cloudprovider/providers/gce/gce.go [hunk 1]
```diff
@@ -188,7 +188,7 @@ func newGCECloud(config io.Reader) (*GCECloud, error) {
 	if err != nil {
 		return nil, err
 	}
-	networkURL := gceNetworkURL(apiEndpoint, projectID, networkName)
+	networkURL := gceNetworkURL("", projectID, networkName)
```
Comment (nicksardo) [inline pkg/cloudprovider/providers/gce/gce.go]: There are a few other calls to `gceNetworkURL` - why do they get apiEndpiont instead of ""?

**Comment by mml** [comment]
Merging since this is blocking the queue.

**Pull request merged**